prod-sample/sample-modules/franks-burgers/0.0.0/src/components/index.jsx

```diff
@@ -0,0 +1,166 @@
+/*
```

Comment:
this example seems a lot more complex than it should be

Reply:
I will simplify the example

Reply:
I have organized the example which I hope reduces complexity - it's meant to be a concise example of code-splitting usage with different approaches and closer to real world scenario.

Reply:
can we get rid of everything that is not related to the code splitting aspect ? we need these modules to be as lightweight as possible.

Reply:
agreed. this module is way too big and has a lot of irrelevant stuff in it. also please dont use this pattern for chunk loading. just react.lazy will do

Reply:
The `sample-module` has been updated and simplified drastically.